### ISSUE TITLE
Function removeWifiNetwork is broken

### ISSUE THREAD

**Issue by csanz91** [alternadom/WiFiFlutter#17]
The argument used in the function removeWifiNetwork: https://github.com/alternadom/WiFiFlutter/blob/4dbbad817981b65f780acdb0ffeac14dcec99e68/lib/wifi_iot.dart#L310 doesn't match the argument used in the .java and .swift side: https://github.com/alternadom/WiFiFlutter/blob/4dbbad817981b65f780acdb0ffeac14dcec99e68/android/src/main/java/com/alternadom/wifiiot/WifiIotPlugin.java#L651.

Changing the argument name to 'prefix_ssid' fixes the issue.

**Comment by alternadom** [comment]
Hi @csanz91 

I think you are refering to a method for retrieving SSID which start with the same prefix, don't you ?

**Comment by alternadom** [comment]
On iOS, if you connect to a WiFi network with an app, you can remove it from the storage.. but that's not possible for WiFi network which was connect from "Settings" on iOS..
So, with this method, you can "purge" all WiFi network created by your apps!

**Comment by csanz91** [comment]
Hello, I have opened two issues because I saw two different problems with the removeWifiNetwork method.
The issue I'm talking about here is related with the name of the argument passed in the method removeWifiNetwork located in the file wifi_iot.dart: https://github.com/alternadom/WiFiFlutter/blob/4dbbad817981b65f780acdb0ffeac14dcec99e68/lib/wifi_iot.dart#L310 
It doesn't correspond with the argument name expected by the method removeWifiNetwork located in the file WifiIotPlugin.java:
https://github.com/alternadom/WiFiFlutter/blob/4dbbad817981b65f780acdb0ffeac14dcec99e68/android/src/main/java/com/alternadom/wifiiot/WifiIotPlugin.java#L651

As the argument expected cannot be found an exception is raised. Changing the name of the argument fixes the issue and the remove method is called successfully.
